charts/kong/example-values/README.md

```diff
@@ -23,6 +23,14 @@ change this value to `true` if you use Helm 2).
   installs Kong for Kubernetes with Kong Enterprise with the ingress controller
   in PostgreSQL. It enables all Enterprise services.
 
+* [minimal-kong-hybrid-control.yaml](minimal-kong-hybrid-control.yaml) and 
+  [minimal-kong-hybrid-data.yaml](minimal-kong-hybrid-data.yaml) install
+  separate releases for hybrid mode control and data plane nodes, using
+  the built-in PostgreSQL chart on the control plane release. They require
+  some pre-work to create certificates and configure the control plane
+  location. See comments in the file headers for details. Note that you should
+  install the control plane release before the data plane release.
```

Comment:
> Note that you should install the control plane release before the data plane release.

1. this is potentially problematic - not all ways to install Helm releases will guarantee that, especially if helm releases are managed by GitOps. For example, [helm-operator](https://github.com/fluxcd/helm-operator) won't allow you to install helm releases in a specific order.

2. consider making this requirement more visible if it's important for deployment success. Possibly as a clear warning in the "Hybrid mode" section.

Reply:
Useful to do so, but not required--it's mostly to make sure the DP nodes are able to become live and ready, since they need to initiate communication with a CP node to begin handling traffic. They do retry if that fails, so this is mostly guidance to bring them up more smoothly.

Adding clarifying notes that explain why you should do that.